app/assets/stylesheets/components/_reset.css.sass

```diff
@@ -3,9 +3,7 @@
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0)
   -webkit-font-smoothing: antialiased
 
-button,
-textarea,
-input[type="text"]
```

Comment:
Because they are't doing anything right now and they remove the border from the inputs which I need.
You only need the rule for the button.